package.json

```diff
@@ -54,7 +54,7 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^3.4.0",
     "jsdom": "^16.6.0",
-    "json5": "^2.2.0",
+    "jsonc-parser": "^3.0.0",
```

Comment:
This is seemingly the parser used by VSCode for "JSON with Comments" parsing mode.